package.json

```diff
@@ -114,7 +114,13 @@
     "webpack-cli": "^3.3.6"
   },
   "peerDependencies": {
-    "webpack": "^4.0.0"
+    "webpack": "^4.0.0",
+    "webpack-cli": "^3.0.0"
```

Comment:
Please remove `webpack-cli` from here, this line break CFA, vue-cli, angular-cli and etc big boilerplates